addon/ng2/tasks/install.js

```diff
@@ -0,0 +1,178 @@
+/* jshint node: true, esnext: true */
```

Comment:
@hansl I would need some of your help here. 
There are/will be actually more possible procedures, for example just `ng install` to install all the packages in the config file. The install procedure can go in many ways according to the result the previous task responses. I am not sure how to achieve that with promises that I have now and I don't want to repeat calling same task multiple times. 
Hope this description of the problem I am trying to avoid is clear.